contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -91,24 +93,19 @@ contract WithdrawalQueue {
     /// - N after upgrading contract from the previous version (after calling finalize_vN())
     bytes32 internal constant CONTRACT_VERSION_POSITION = keccak256("lido.WithdrawalQueue.contractVersion");
 
-    /// Lido DAO Agent contract address
-    /// Used to call administrative levers
-    bytes32 internal constant LIDO_DAO_AGENT_POSITION = keccak256("lido.WithdrawalQueue.lidoDAOAgent");
-
-    /// Requests placement resume/pause control storage slot
-    bytes32 internal constant REQUESTS_PLACEMENT_RESUMED_POSITION =
-        keccak256("lido.WithdrawalQueue.requestsPlacementResumed");
+    /// Withdrawal queue resume/pause control storage slot
+    bytes32 internal constant RESUMED_POSITION = keccak256("lido.WithdrawalQueue.resumed");
```

Comment:
Claims should be paused together with placements, but this short name is totally ok (speaking about the previous convos).

Reply:
Ok. Let's do the safe way at first. But from my point of view, it's not fair to pause already locked ether.